README.md

```diff
@@ -16,6 +16,13 @@ RapiDAST provides:
 
 RapiDAST is for testing purposes, and should not be used on production systems.
 
+## Deprecation Notice
+
+**Podman Mode Deprecation**
+The `podman` execution environment is deprecated and will be removed in version **2.12**
+
+If you are using `podman`, please migrate to `none` before updating to version 2.12.
```

Comment:
For better clarity, how about "If you are using `podman` for the `container.type` option"?